src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-decoration: none;
+    color: #060b35;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    font-family: inherit;
+    font-size: inherit;
+    font-weight: 400;
```

Comment:
This is the default value, you don't need to specify it.
```suggestion
```

Reply:
Also, please fix these comments